ports/rp2/mpconfigport.h

```diff
@@ -116,6 +116,7 @@
 #if MICROPY_HW_USB_MSC
 #define MICROPY_FATFS_USE_LABEL                 (1)
 #define MICROPY_FATFS_MULTI_PARTITION           (1)
+#define MICROPY_HW_USB_MSC_EXCLUSIVE_ACCESS     (1)
```

Comment:
If MICROPY_HW_USB_MSC_EXCLUSIVE_ACCESS means exclusive write access, the this must be the default state. Everything else is just risky. The exclusive write state of the MSC will be terminated by ejecting the disk. That's fine and simple, and avoids problems with differing views of the file system.
So it may not be necessary to have a #define for it.

Reply:
Yes it can be enabled by default, but it's good to have the option to disable it if it's not needed.